synapse/storage/data_stores/state/schema/deltas/32/remove_indices.sql

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2016 OpenMarket Ltd
```

Comment:
won't having two delta files with the same name mean that only one gets run? or am I misremembering how this works?

Reply:
Oh, you're right. I might add a sanity check to cause synapse to explode if there are multiple deltas with the same name